js/shims.d.ts

```diff
@@ -13,7 +13,7 @@ import Application from './src/common/Application';
  * Helpers that flarum/core patches into Mithril
  */
 interface m extends Mithril.Static {
-  prop: typeof Stream;
+  stream: typeof Stream;
```

Comment:
Is this still needed after #2316 ? I think `m.prop` is only there for BC with beta13 and `m.stream` is only there for BC with master extensions that haven't been updated yet? So this should no longer be needed with beta14?

Reply:
yeah that can be removed